File: matlab/getBB3D.m
function bbObjCord = getBB3D(maxExtent)

bbminX = -maxExtent(1)/2;
bbmaxX =  maxExtent(1)/2;

bbminY = -maxExtent(2)/2;
bbmaxY =  maxExtent(2)/2;

bbminZ = -maxExtent(3)/2;
bbmaxZ =  maxExtent(3)/2;

v1 = [ bbminX bbminY bbminZ ];
v2 = [ bbmaxX bbminY bbminZ ];
v3 = [ bbmaxX bbmaxY bbminZ ];
v4 = [ bbminX bbmaxY bbminZ ];

v5 = [ bbminX bbminY bbmaxZ ];
v6 = [ bbmaxX bbminY bbmaxZ ];
v7 = [ bbmaxX bbmaxY bbmaxZ ];
v8 = [ bbminX bbmaxY bbmaxZ ];

bbObjCord = [v1; v2 ;v3; v4 ;v5; v6 ;v7 ;v8];

File: python/getBB3D.py
import numpy as np

def getBB3D(maxExtent):

    bbminX = -maxExtent[0]/2;
    bbmaxX =  maxExtent[0]/2;
    
    bbminY = -maxExtent[1]/2;
    bbmaxY =  maxExtent[1]/2;
    
    bbminZ = -maxExtent[2]/2;
    bbmaxZ =  maxExtent[2]/2;

    bbObjCord = np.array([
         bbminX, bbminY, bbminZ,
         bbmaxX, bbminY, bbminZ,
         bbmaxX, bbmaxY, bbminZ,
         bbminX, bbmaxY, bbminZ,
         bbminX, bbminY, bbmaxZ,
         bbmaxX, bbminY, bbmaxZ,
         bbmaxX, bbmaxY, bbmaxZ,
         bbminX, bbmaxY, bbmaxZ]).reshape([8,3]);
    
    return bbObjCord